<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MatrixCal</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>3</number>
       </property>
       <widget class="MatrixEditorPersistent" name="MatrixEditor">
        <attribute name="title">
         <string>Matrix Editor</string>
        </attribute>
       </widget>
       <widget class="MatrixCalculationPersistent" name="MatrixCalculation">
        <attribute name="title">
         <string>Matrix Calculation</string>
        </attribute>
       </widget>
       <widget class="EquationSolverPersistent" name="LinearSolver">
        <attribute name="title">
         <string>LinearSystem</string>
        </attribute>
       </widget>
       <widget class="BinaryCalculatorPersistent" name="BinaryCalculator">
        <attribute name="title">
         <string>Binary Calculator</string>
        </attribute>
       </widget>
      </widget>
      <widget class="SearchListView" name="listView" native="true"/>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QListView" name="StatesList"/>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>Clear History</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuEditor">
    <property name="title">
     <string>Editor</string>
    </property>
    <addaction name="actionAdd_to_list"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionPreferences"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuEditor"/>
  </widget>
  <action name="actionAdd_to_list">
   <property name="text">
    <string>Store matrix</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>Preferences...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MatrixEditorPersistent</class>
   <extends>QWidget</extends>
   <header>ui.widgets.MatrixEditor.MatrixEditor</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SearchListView</class>
   <extends>QWidget</extends>
   <header>ui.views.SearchListView</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>EquationSolverPersistent</class>
   <extends>QWidget</extends>
   <header>ui.widgets.EquationSolver.EquationSolver</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MatrixCalculationPersistent</class>
   <extends>QWidget</extends>
   <header>ui.widgets.MatrixCalculation.MatrixCalculation</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BinaryCalculatorPersistent</class>
   <extends>QWidget</extends>
   <header>ui.widgets.BinaryCalculator.BinaryCalculator</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>actionPreferences</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>showPreferencesDialog()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>639</x>
     <y>359</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>clearHistory()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>896</x>
     <y>674</y>
    </hint>
    <hint type="destinationlabel">
     <x>890</x>
     <y>692</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>showPreferencesDialog()</slot>
  <slot>clearHistory()</slot>
 </slots>
</ui>
